styles.css

```diff
@@ -0,0 +1,962 @@
+    text-align: center;
+    color: #6070ff;
+  }
+
+  .hamburger {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 12px;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    right: 4%;
+    top: calc(50% - 40px / 2);
+  }
+
+  .bar {
+    display: block;
+    width: 30px;
+    height: 3px;
+    margin: 3px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #6070ff;
+  }
+
+  #headline {
+    position: absolute;
+    width: 100%;
+    height: 587.15px;
+    left: 0;
+    background-color: #fff;
+    top: 15%;
+    background-image: url("images/header-shapes\ mobile.png");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    border-bottom-left-radius: 50px;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 114px 24px;
+    gap: 2px;
+    position: absolute;
+    width: 375px;
+    height: 604px;
+    left: calc(50% - 375px / 2);
+    top: -10%;
+  }
+
+  .primary-text1 {
+    width: 327px;
+    height: 150px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    margin-bottom: 0%;
+  }
+
+  .primary-text2 {
+    width: 327px;
+    height: 146px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .socialmedia {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 16px;
+    width: 200px;
+    height: 24px;
+  }
+
+  .supporting-text {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    width: 200px;
+    height: 24px;
+  }
+
+  .connect {
+    width: 200px;
+    height: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .socialmedia-icons {
+    width: 170.17px;
+    height: 20px;
+  }
+
+  .twitter {
+    position: absolute;
+    left: 6.4%;
+    right: 88.27%;
+    top: 63.15%;
+    bottom: 18.88%;
+  }
+
+  .linkedin {
+    position: absolute;
+    width: 20px;
+    height: 20px;
+    left: 17.47%;
+    top: 63.15%;
+  }
+
+  .m {
+    position: absolute;
+    left: 27.47%;
+    right: 67.2%;
+    top: 63.15%;
+    bottom: 19.21%;
+  }
+
+  .github {
+    position: absolute;
+    left: 38.13%;
+    right: 57.07%;
+    top: 63.15%;
+    bottom: 18.87%;
+  }
+
+  .vector {
+    position: absolute;
+    left: 48%;
+    right: 48.22%;
+    top: 63.15%;
+    bottom: 18.87%;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```

Comment:
- [ ] Kindly use `display: grid` instead of `display: flex` because this is a requirement check below:

![Screenshot from 2023-02-17 17-14-50](https://user-images.githubusercontent.com/45226848/219651222-d8462a46-c777-4192-9f78-1ba51cbda11a.png)